edx_name_affirmation/views.py

```diff
@@ -106,3 +123,47 @@ def post(self, request):
             response_status = status.HTTP_400_BAD_REQUEST
             data = serializer.errors
         return Response(status=response_status, data=data)
+
+
+class VerifiedNameConfigView(AuthenticatedAPIView):
+    """
+    Endpoint for VerifiedNameConfig.
+    /edx_name_affirmation/v1/verified_name/config
+
+    Supports:
+        HTTP POST: Creates a new VerifiedNameConfig.
+
+    HTTP POST
+    Creates a new VerifiedName.
+    Example POST data: {
+        "username": "jdoe",
+        "use_verified_name_for_certs": True
+    }
+    """
+    def post(self, request):
+        """
+        Create VerifiedNameConfig
+        """
+        username = request.data.get('username')
+        if username != request.user.username and not request.user.is_staff:
+            return Response(
+                status=status.HTTP_403_FORBIDDEN,
+                data={"detail": "Must be a Staff User to Perform this request."}
```

Comment:
Can you be more descriptive on this error detail? Like `Must be a staff user to override the requested user's VerifiedNameConfig value`?